{
	"id": "oval:org.debian:def:332484745479400073073497896019366453405",
	"class": "patch",
	"metadata": {
		"title": "DSA-5018-1 python-babel",
		"affected": {
			"family": "unix",
			"platform": "Debian GNU/Linux 10",
			"product": "python-babel"
		},
		"reference": [
			{
				"source": "CVE",
				"ref_id": "CVE-2021-42771",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42771"
			}
		],
		"description": "security update",
		"debian": {
			"moreinfo": "no info",
			"dsa": "DSA-5018",
			"date": "2021-12-09"
		}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterias": [
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.debian.oval:tst:2",
								"comment": "all architecture"
							},
							{
								"test_ref": "oval:org.debian.oval:tst:28402",
								"comment": "python-babel DPKG is earlier than 2.6.0+dfsg.1-1+deb10u1"
							}
						]
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.debian.oval:tst:1",
				"comment": "Debian 10 is installed"
			}
		]
	}
}
